src/win/poll.c

```diff
@@ -91,7 +91,8 @@ static void uv__fast_poll_submit_poll_req(uv_loop_t* loop, uv_poll_t* handle) {
     handle->mask_events_1 = handle->events;
     handle->mask_events_2 = 0;
   } else {
-    assert(0);
+    /* Just wait until there's an unsubmitted req. */
+    /* It will be almost immediately */
```

Comment:
Yeah, I see it's not clear at all. Sorry about that. Is this better?
```
/* Just wait until there's an unsubmitted req. */
/* This will happen almost immediately as one of the 2 outstanding */
/* requests is about to return */
```